middleware/swagger-metadata.js

```diff
@@ -225,7 +225,7 @@ var processOperationParameters = function (swaggerMetadata, pathKeys, pathMatch,
 
   debug('  Processing Parameters');
 
-  async.map(_.reduce(parameters, function (requestParsers, parameter) {
+  var parsers = _.reduce(parameters, function (requestParsers, parameter) {
```

Comment:
Explanation makes sense to me.  Let me take one more peek and we'll go from there.